ui/src/components/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React, {Component} from 'react';
+import {FormattedMessage, defineMessages, injectIntl} from 'react-intl';
+
+const messages = defineMessages({
+  disclaimerText: {
+    id: 'Footer.disclaimerText',
+    defaultMessage: ('PLEASE NOTE: all data entered is available to the public '
+        + 'and usable by anyone. Google does not review or verify the accuracy '
+        + 'of this data. Google may share the data with public and private '
+        + 'organizations participating in disaster response efforts.'),
+    description: 'A disclaimer shown at the footer of the page.',
+  },
+});
+
+class Footer extends Component {
```

Comment:
Optional: This can be a simple function instead of a class as you did for App component:

```jsx
const Footer = () => (
  <div>...</div>
);
```

Reply:
Done.